examples/concurrency/lib/simple_isolate_closure.dart

```diff
@@ -0,0 +1,20 @@
+import 'dart:convert';
+import 'dart:io';
+import 'dart:isolate';
+
+const String filename = 'with_keys.json';
+
+// #docregion main
+void main() async {
+  // Read some data.
+  final jsonData = await Isolate.run(() {
+    final fileData = await File(filename).readAsString();
+    final jsonData = jsonDecode(fileData) as Map<String, dynamic>; 
+    return await jsonData;
+  });
+  
+  // Use that data
```

Comment:
Extreme nit: Put a `.` at the end (Dart style says format comments as sentences).